CSSS/Kernel.cs

```diff
@@ -172,28 +167,34 @@ public bool PerformTasks()
             }
 
             // If we should shutdown
-            if (shouldShutdown)
+            if (config.CSSSProgramMode.HasFlag(Config.CSSSModes.Shutdown))

```

Comment:
Could this be an `if` statement containing checks for both `shutdown` and `prepare`? This way it will only shutdown the computer when `-p` is used.